### PR TITLE
Extend DCs and add timeouts for resources, 

### PR DESCRIPTION
**Pull request by surajsbharadwaj**
### Description

Please include a summary of the change and which issue is fixed. Also, please include the motivation and context for the change. List any dependencies that are required for this change.

**Check the relevant boxes:**
- [x] Bug fix (nonbreaking change that fixes an issue)
- [ ] New feature (nonbreaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Examples or tests (addition or updates of examples or tests)
- [ ] Documentation update
- [ ] CI related update (pipeline, etc.)

### Checklist

- [ ] If relevant, a test for the change has been added or updated as part of this PR.
- [ ] If relevant, documentation for the change has been added or updated as part of this PR.

### Merge

- Merge using "Squash and merge".
- Make sure to use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents. The commit message determines whether a new version of the modules needs to be released, and if so, which semver number to use).

### CODE REVIEW

File: examples/basic/variables.tf [hunk 1]
```diff
@@ -1,5 +1,5 @@
 variable "powervs_zone" {
-  description = "IBM Cloud data center location where IBM PowerVS infrastructure will be created. Following locations are currently supported: syd04, syd05, eu-de-1, eu-de-2, tok04, osa21, sao01"
+  description = "IBM Cloud data center location where IBM PowerVS infrastructure will be created. Following locations are currently supported: syd04, syd05, eu-de-1, eu-de-2, tok04, osa21, sao01, lon04, lon06"
```
Comment (ocofaigh) [inline examples/basic/variables.tf:2]: how about adding a validation check to ensure only supported values are passed (https://developer.hashicorp.com/terraform/language/values/variables#custom-validation-rules)

Comment (ocofaigh) [reply examples/basic/variables.tf:2]: oh I see you have it in the main modules variables.tf, so maybe not needed here too

File: examples/standard/variables.tf [hunk 2]
```diff
@@ -5,7 +5,7 @@ variable "ibmcloud_api_key" {
 }
 
 variable "powervs_zone" {
-  description = "IBM Cloud data center location where IBM PowerVS infrastructure will be created. Following locations are currently supported: syd04, syd05, eu-de-1, eu-de-2, tok04, osa21, sao01"
+  description = "IBM Cloud data center location where IBM PowerVS infrastructure will be created. Following locations are currently supported: syd04, syd05, eu-de-1, eu-de-2, tok04, osa21, sao01, lon04, lon06"
```
[the remark this answers is not in the document]
Comment (ocofaigh) [reply examples/standard/variables.tf:8]: oh I see you have it in the main modules variables.tf, so maybe not needed here too

File: examples/basic/variables.tf (hunk 1 above)
Comment (surajsbharadwaj) [reply examples/basic/variables.tf:2]: Right

**Pull request closed without merging**